modules/oe_theme_content_event/src/Plugin/PageHeaderMetadata/EventContentType.php

```diff
@@ -39,7 +39,7 @@ public function getMetadata(): array {
       ->getOptionsProvider('value', $node);
 
     $metadata['metas'] = [
-      $provider->getPossibleOptions()[$node->get('oe_event_type')->value],
+      $provider->getPossibleOptions()[$node->get('oe_event_type')->target_id],
```

Comment:
This is wrong here, we need to render the label as the field type is not `list_string` anymore, check how we do it here with a similar field type `\Drupal\oe_theme_content_event\Plugin\ExtraField\Display\DetailsExtraField::getRenderableSubject`